packages/expo-mail-composer/src/MailComposer.ts

```diff
@@ -1,15 +1,20 @@
 import ExpoMailComposer from './ExpoMailComposer';
 import { MailComposerOptions, MailComposerResult } from './MailComposer.types';
 
+/**
+ * Opens a mail modal for iOS and a mail app intent for Android and fills the fields with provided data. On iOS you will need to be signed into the Mail app.
+ * @return Resolves to a promise with object containing status field that could be either sent, saved or cancelled. Android does not provide such info so it always resolves to sent.
```

Comment:
Now that this documentation is extra visible, technical correctness and quality of writing matter even more:

```suggestion
 * @return A promise fulfilled with an object containing a `status` field that specifies whether an email was sent, saved, or cancelled. Android does not provide this info, so the status is always set as if the email were sent.
```

(@brentvatne I think we will need higher expectations for authors and reviewers for our docblocks now. High-quality docs reflect upon our team's competence.)